packages/canvas/canvas-graphics/src/CanvasGraphicsRenderer.ts

```diff
@@ -73,6 +73,118 @@ export class CanvasGraphicsRenderer
         return res;
     }
 
+    /**
+     * draw shape on canvas
+     *
+     * @private
+     * @param {PIXI.FillStyle} fillStyle
+     * @param {PIXI.LineStyle} lineStyle
+     * @param {string|CanvasPattern} contextFillStyle
+     * @param {string|CanvasPattern} contextStrokeStyle
+     * @param {number} worldAlpha
+     */
+    private draw(
```

Comment:
I dunno if we should call this `paint` (fill + stroke = paint) since `draw` is for actually pushing a primitive/shape into the rendering pipeline (cc @bigtimebuddy)

Reply:
It's private, so it doesn't matter to me. If it were public, I'd scrutinize this more. `paint` is fine too.